great_expectations/expectations/metrics/map_metric.py

```diff
@@ -803,12 +796,12 @@ def _pandas_map_condition_rows(
 
     result_format = metric_value_kwargs["result_format"]
 
+    df = df[boolean_mapped_unexpected_values]
+
     if result_format["result_format"] == "COMPLETE":
```

Comment:
Is there any chance of `result_format` missing the "result_format" key? Should we make this a `get()`?